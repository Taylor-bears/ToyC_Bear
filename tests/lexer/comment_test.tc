int main() {
  /*
    This is a comment.
    It contains a fake end token:  * /  (with a space)
    It also contains fake code: if (a > b) return;
  */
  return /* inline comment */ 1;
}