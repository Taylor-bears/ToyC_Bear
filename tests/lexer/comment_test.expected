INT
ID(main)
LPAREN
RPAREN
LBRACE
RETURN
NUMBER(1)
SEMICOLON
RBRACE
EOF